### ISSUE TITLE
Exposure of Sensitive Information in eventsource

### ISSUE THREAD

**Issue by KieraDOG** [splitio/javascript-client#677]
Exposure of Sensitive Information in eventsource

The latest possible version of eventsource that can be installed is 1.1.1. The earliest fixed version is 2.0.2.

```
  "optionalDependencies": {
    "eventsource": "^1.0.7"
  },
```

https://github.com/advisories/GHSA-6h5x-7c5m-7cr7

**Comment by DaleGardner** [comment]
This needs #678 to resolve

**Comment by chillaq** [comment]
Thanks @KieraDOG , checking with the team.

**Comment by EmilianoSanchez** [comment]
Hi @KieraDOG. We are releasing the fix next week

**Comment by NicoZelaya** [comment]
Hi @KieraDOG ! 

Would you mind trying with installing the latest version of the SDK and re-validating if the vulnerability keeps appearing? 
This problem is also fixed in eventsource version 1.1.1, already used by the SDK, as you can see in this link from the advisory on Github: https://github.com/advisories/GHSA-6h5x-7c5m-7cr7 

If you still get it reported would you mind sharing more with us? We'll release this breaking change eventually, but since it has no value for this fix, it might not be worth waiting for it.

Apologies for the inconveniences,
Nicolas Zelaya.

**Comment by KieraDOG** [comment]
Hi @NicoZelaya `eventsource@1.1.1` fixed the problem
